sequencing/clinical_annotator/resources/home/dnanexus/ClinVar_tsvTOvcf.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+"""Script to reformat the ClinVar TSV file into two VCF files, one for b37 and
+one for b38.
+"""
+__author__ = "Thomas Nate Person"
+__license__ = "GPLv3"
+__email__ = "thomas.n.person@gmail.com"
+
```

Comment:
It's safer to do VCF parsing using pyvcf or cyvcf, but I understand if you don't want to since that's a fairly major rewrite.